### ISSUE TITLE
Not launching latest Netbeans 10 on MacOS 10.14.3

### ISSUE THREAD

**Issue by robross0606** [carljmosca/netbeans-macos-bundle#28]
The app file appears in the correct location.  Launching it starts to launch, the "Netbeans" menu appears for a second or two at the top and then the app crashes out.  No error message appears.  Not sure how to get logs from this problem.

**Comment by robross0606** [comment]
All I've found so far this this in my system.log:
```
Mar 18 10:19:22 Roberts-Mac com.apple.xpc.launchd[1] (org.netbeans.ide.baseide.10.8288[8159]): Service exited with abnormal code: 1
```

**Comment by Polarisation** [comment]
I'm seeing the same or a very similar issue when opening the newly installed Netbeans 10 on MacOS 10.14.3, Netbeans 10. Just a different exit code:

```
com.apple.xpc.launchd[1] (org.netbeans.ide.baseide.10.6528[2722]): Service exited with abnormal code: 2
```

This is the first time I attempted to install NetBeans on this Mac.

**Comment by walterleonardo** [comment]
Same problem for me.

**Comment by ulab** [comment]
Do you have Java / the JDK installed like the comment above yours suggested?

**Comment by robross0606** [comment]
Yes, I had the JDK installed.  I replaced a previous version of Netbeans which was working fine.
